client/my-sites/marketing/tools/index.tsx

```diff
@@ -52,6 +54,10 @@ export const MarketingTools: FunctionComponent = () => {
 		recordTracksEvent( 'calypso_marketing_tools_create_a_logo_button_click' );
 	};
 
+	const handleLetsGoClick = () => {
+		recordTracksEvent( 'calypso_marketing_tools_lets_go_button_click' );
```

Comment:
Great adding Tracks events in! I'd probably name it something related directly to promoted posts. Eg. if we in the future have another button saying Let's go, or if we change the button text in the future. Having a more clear name, including promoted posts, makes it much easier to run reports. :)

Reply:
Updated in code.